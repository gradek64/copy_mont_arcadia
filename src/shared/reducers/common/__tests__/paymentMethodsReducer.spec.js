import testReducer from '../paymentMethodsReducer'

describe('Payment Method Reducer', () => {
  describe('SET_PAYMENT_METHODS', () => {
    it('stores payload in state', () => {
      const newState = testReducer([], {
        type: 'SET_PAYMENT_METHODS',
        payload: ['foo'],
      })

      expect(newState).toMatchSnapshot()
      expect(newState).toEqual(['foo'])
    })
  })
})
