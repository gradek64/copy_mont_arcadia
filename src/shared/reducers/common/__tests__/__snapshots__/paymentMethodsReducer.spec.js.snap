// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Payment Method Reducer SET_PAYMENT_METHODS stores payload in state 1`] = `
Array [
  "foo",
]
`;
